### DISCUSSION TITLE
Is it possible to specify the vercel project to deploy to?

### DISCUSSION THREAD

**Discussion by 0xAlwaysDumpling** [vercel/community#4524] [Help] [answered]
### Summary

I have a repo where each branch is a different vercel project (maps to a sub domain). Is is possible to specify that so that whenever I push to a branch it doesn't build to both projects?

### Steps to Reproduce

1. Single repo with vercel projects for 2 different branches
2. When we push a commit to a branch, vercel will build for both projects. 
3. I have to manually cancel the deployment of one of the projects.

**Comment by amyegan** [comment, marked as answer]
It sounds like the [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) feature is what you need. That will let you configure the conditions that should be met before a project is automatically deployed. 

https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel
